module/config.mjs

```diff
@@ -112,31 +121,32 @@ DND5E.hitPointsAbility = "con";
  * @typedef {object} SkillConfiguration
  * @property {string} label    Localized label.
  * @property {string} ability  Key for the default ability used by this skill.
+ * @property {string} fullKey  Fully written key used as alternate for enrichers.
  */
 
 /**
  * The set of skill which can be trained with their default ability scores.
  * @enum {SkillConfiguration}
  */
 DND5E.skills = {
-  acr: { label: "DND5E.SkillAcr", ability: "dex" },
-  ani: { label: "DND5E.SkillAni", ability: "wis" },
-  arc: { label: "DND5E.SkillArc", ability: "int" },
-  ath: { label: "DND5E.SkillAth", ability: "str" },
-  dec: { label: "DND5E.SkillDec", ability: "cha" },
-  his: { label: "DND5E.SkillHis", ability: "int" },
-  ins: { label: "DND5E.SkillIns", ability: "wis" },
-  itm: { label: "DND5E.SkillItm", ability: "cha" },
-  inv: { label: "DND5E.SkillInv", ability: "int" },
-  med: { label: "DND5E.SkillMed", ability: "wis" },
-  nat: { label: "DND5E.SkillNat", ability: "int" },
-  prc: { label: "DND5E.SkillPrc", ability: "wis" },
-  prf: { label: "DND5E.SkillPrf", ability: "cha" },
-  per: { label: "DND5E.SkillPer", ability: "cha" },
-  rel: { label: "DND5E.SkillRel", ability: "int" },
-  slt: { label: "DND5E.SkillSlt", ability: "dex" },
-  ste: { label: "DND5E.SkillSte", ability: "dex" },
-  sur: { label: "DND5E.SkillSur", ability: "wis" }
+  acr: { label: "DND5E.SkillAcr", ability: "dex", fullKey: "acrobatics" },
+  ani: { label: "DND5E.SkillAni", ability: "wis", fullKey: "animalHandling" },
+  arc: { label: "DND5E.SkillArc", ability: "int", fullKey: "arcana" },
+  ath: { label: "DND5E.SkillAth", ability: "str", fullKey: "athletics" },
+  dec: { label: "DND5E.SkillDec", ability: "cha", fullKey: "deception" },
+  his: { label: "DND5E.SkillHis", ability: "int", fullKey: "history" },
+  ins: { label: "DND5E.SkillIns", ability: "wis", fullKey: "insight" },
+  itm: { label: "DND5E.SkillItm", ability: "cha", fullKey: "intimidation" },
+  inv: { label: "DND5E.SkillInv", ability: "int", fullKey: "investigation" },
+  med: { label: "DND5E.SkillMed", ability: "wis", fullKey: "medicine" },
+  nat: { label: "DND5E.SkillNat", ability: "int", fullKey: "nature" },
+  prc: { label: "DND5E.SkillPrc", ability: "wis", fullKey: "perception" },
+  prf: { label: "DND5E.SkillPrf", ability: "cha", fullKey: "performance" },
+  per: { label: "DND5E.SkillPer", ability: "cha", fullKey: "persuasion" },
+  rel: { label: "DND5E.SkillRel", ability: "int", fullKey: "religion" },
+  slt: { label: "DND5E.SkillSlt", ability: "dex", fullKey: "slightOfHand" },
+  ste: { label: "DND5E.SkillSte", ability: "dex", fullKey: "stealth" },
+  sur: { label: "DND5E.SkillSur", ability: "wis", fullKey: "survival" }
```

Comment:
These keys will not be replaced by localization, otherwise existing enrichers that point to them would break as soon as you enable a new localization.